server.js

```diff
@@ -2048,12 +2048,18 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // iTunes App Store version
-camp.route(/^\/itunes\/v\/(.+)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/itunes\/v\/(?:(.+)\/)?(.+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var bundleId = match[1];  // eg, `324684580`
-  var format = match[2];
-  var apiUrl = 'https://itunes.apple.com/lookup?id=' + bundleId;
+  var region = match[1]; // eg, `us`
+  var bundleId = match[2];  // eg, `324684580`
+  var format = match[3];
+
+  var apiUrl = (region == '')
```

Comment:
Can you edit this to use a strict comparison here (triple equals).

Also I know they were `var` when you started, but while you're editing this block of code, could you update these variable declarations to use the ES6 style `const` please.